AdyenDropIn/DropInComponent.swift

```diff
@@ -238,7 +240,7 @@ public final class DropInComponent: NSObject, PresentableComponent {
         
         if isRoot {
             self.delegate?.didFail(with: ComponentError.cancelled, from: self)
-        } else {
+        } else if component.requiresModalPresentation {
```

Comment:
Don't we need to add an 'else' case still?

Reply:
I don't think so cause we only pop on the navigation controller if the component is presented modally

Reply:
So, when cancelling non-modal presentation (like SFViewController, PKApplePayViewConrtoller etc) `requiresModalPresentation` is clicked we are going to do nothing?

Reply:
After off chat call with @descorp, reverting this change.